src/sympc/tensor/mpc_tensor.py

```diff
@@ -736,6 +758,37 @@ def _get_shape(
         res = op(x, y, **kwargs_)
         return res.shape
 
+    def truncation(
+        self, input_tensor: "MPCTensor", op_str: str, is_private: bool
+    ) -> "MPCTensor":
+        """Checks if operation requires truncation and performs it if required.
+
+        Args:
+            input_tensor (MPCTensor): Result of operation
+            op_str (str): Operation name
+            is_private (bool): If operation is private
+
+        Returns:
+            result (MPCTensor): Truncated result
+        """
+        # Truncation should be added for RSTensor
+        if (
+            op_str in {"mul", "matmul", "conv2d", "conv_transpose2d"}
```

Comment:
Could we keep this set in a more accessible place as we might be adding more to it?

Something like these: 

https://github.com/OpenMined/SyMPC/blob/94fd2f3a440d0bad4a289a1fdece9b436fa34d5a/src/sympc/tensor/mpc_tensor.py#L29

Reply:
Hmmm,  I am not sure if adding it to an accessible place would help since this is the only function we will be using it in, and can't think of a common name for grouping them. But, in general, I think this whole MPCclass needs refactoring in a future PR.